play-media-chone/01. Content Management/03. Create Content Types/README.md

```diff
@@ -0,0 +1,164 @@
+# Create Content
+
+This scenario demonstrates creating a new content type, creating and publishing new pieces of content in Content Hub ONE and displaying them on a new page of the website.
+
+1. Log into the [Sitecore Cloud Portal](https://portal.sitecorecloud.io/) to access your Content Hub ONE instances.
+
+2. Navigate to your Content Hub ONE instance using the left (top corner) navigation menu and select the "Manage content types" link option.
+   ![Cloud Portal Navigation](./media/content-types-01.png)
+
+3. Create a new content type by clicking on the **"+ Add type"** button in the top right corner.
+   ![Add New Content Type](./media/content-types-02.png)
+
+4. Name the new content item **"Content Block"**. Make sure the id field that gets filled out automatically matches the screenshot below, if not rename it to match.
+   ![Add Content Type](./media/content-types-03.png)
+
+5. Click the **"+ Add field"** button and select **Text**. Name the text field **"Title"**. Make sure the id field that gets filled out automatically matches the screenshot below, if not rename it to match. For Text type select "Long Text".
+   ![Create Title Field](./media/content-types-04.png)
```

Comment:
For the next times you take documentation screenshots, please try to avoid left/right side empty space in screenshots by either:
- Zooming your screen
- Resizing the browser window

This will help with the readability of the documentation.
We can leave the screenshots as is for this PR.